docs/development/setup.md

```diff
@@ -0,0 +1,25 @@
+# Setup
+
+Ensure you have the following installed first:
+
+- librocksdb-dev
+- libclang-dev
+- clang lldb lld
+- build-essential
+- protoc
+
+You will also need to install rust.
+
+To install Rust:
+
+```
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```

Comment:
When I tried to build the chain after installing rust with this command, I got the following error "Rust nightly not installed, please install it!". So you should add a command for switching to the nightly version after installing it.